tests/rateArticle.test.js

```diff
@@ -0,0 +1,66 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../src/index';
+import statuses from '../src/helpers/constants/status.codes';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+let userToken = '';
+
+describe('test the rating controller', () => {
+  it('should signup a user to be checking against on rating article', (done) => {
+    chai
+      .request(app)
+      .post('/api/users/signup')
+      .send({
+        username: 'Premices',
+        email: 'Premices23@gmail.com',
+        password: 'MUfra123qwe'
+      })
+      .end((err, res) => {
+        userToken = res.body.user.token;
+        done();
+      });
+  });
+  it('should login a user and get a token to use against on rating article', (done) => {
+    chai
+      .request(app)
+      .post('/api/users/login')
```

Comment:
why login and yet you have the token on register??

Reply:
Yeah you are right there is no reason to have both, I am on it now